src/containers/Review/assignment/AssignmentRows.tsx

```diff
@@ -14,38 +16,54 @@ interface AssignmentRowsProps {
   fullStructure: FullStructure
   assignmentInPreviousStage: AssignmentDetails
   assignmentGroupedLevel: LevelAssignments
-  assignedSections: SectionAssignee
-  setAssignedSections: React.Dispatch<SetStateAction<SectionAssignee>>
+  assignedSectionsByLevel: AssignedSectionsByLevel
+  setAssignedSectionsByLevel: React.Dispatch<SetStateAction<AssignedSectionsByLevel>>
   setEnableSubmit: React.Dispatch<SetStateAction<boolean>>
   setAssignmentError: React.Dispatch<SetStateAction<string | null>>
 }
 const AssignmentRows: React.FC<AssignmentRowsProps> = ({
   fullStructure,
   assignmentInPreviousStage,
   assignmentGroupedLevel,
-  assignedSections,
-  setAssignedSections,
+  assignedSectionsByLevel,
+  setAssignedSectionsByLevel,
   setEnableSubmit,
   setAssignmentError,
 }) => {
   const { reviewStructuresState } = useReviewStructureState()
 
+  const defaultAssignedSections = Object.values(fullStructure.sections).reduce(
+    (assignedSections, { details: { code } }) => ({
+      ...assignedSections,
+      [code]: { newAssignee: undefined },
+    }),
+    {}
+  )
+
   return (
     <>
       {Object.values(fullStructure.sections).map(({ details: { id, title, code } }) => (
         <Segment key={id}>
           <Header className="section-title" as="h5" content={title} />
           {Object.entries(assignmentGroupedLevel).map(([level, assignments]) => (
             <div className="flex-column" key={`assignment-group-level-${level}`}>
-              <AssignmentSectionRow
-                assignments={assignments}
-                sectionCode={code}
-                reviewLevel={Number(level)}
-                structure={fullStructure}
-                assignedSectionsState={[assignedSections, setAssignedSections]}
-                setEnableSubmit={setEnableSubmit}
-                setAssignmentError={setAssignmentError}
-              />
+              {fullStructure.info.outcome === ApplicationOutcome.Pending && (
+                <AssignmentSectionRow
+                  assignments={assignments}
+                  sectionCode={code}
+                  reviewLevel={Number(level)}
+                  structure={fullStructure}
+                  assignedSections={assignedSectionsByLevel[level] || defaultAssignedSections}
+                  setAssignedSections={(assignedSections) =>
+                    setAssignedSectionsByLevel({
+                      ...assignedSectionsByLevel,
+                      [level]: assignedSections,
+                    })
+                  }
+                  setEnableSubmit={setEnableSubmit}
+                  setAssignmentError={setAssignmentError}
+                />
+              )}
```

Comment:
Can you merge `develop` into this? I think it will remove some of the stuff that is unrelated

Reply:
Okay, it should be all good now.